### PR TITLE
[Feature] Add support for github enterprise

### PR DESCRIPTION
**Pull request by nlunets**
This Pull Request enable the support for github enterprise with dosco.

I'm just leveraging the basic capabilities of octonode to pass in an hostname for the enterprise. 
If it's not filled in (the default case),it will default to standard github.

Thanks :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ At the end of this sequence of steps you will have:
 * Any dependent modules linked between projects (e.g. if one repo is actually a module depended on by another).
 * All projects fully npm installed.
 
+### Github enterprise support
+
+In order to use Dosco with your github enterprise account an additional parameter is avaialable in the bosco config file.
```
Comment (codeaholics) [inline README.md]: Bosco

**Comment by cliftonc** [comment]
This overlaps perfectly with the new docker for mac changes.

File: README.md (hunk 1 above)
Comment (nlunets) [inline README.md]: My bad :) For some reason I keep wanting to call this Dosco !

**Comment by cliftonc** [comment]
Looks great - should go out today, thanks for contributing!

**Comment by nlunets** [comment]
Thanks for the quick review !

**Pull request merged**